sssom/sparql_util.py

```diff
@@ -84,10 +84,22 @@ def query_mappings(config: EndpointConfig) -> MappingSetDataFrame:
 
 
 def curiefy_row(row: Mapping[str, str], config: EndpointConfig) -> Dict[str, str]:
+    """Assign a contracted URI ("dc") if Mapping object dictionary value is "http://purl.org/dc/terms/".
+
+    :param row: Mapping object row
+    :param config: Configuration
+    :return: Dictionary of CURIEs
+    """
     return {k: contract_uri(v, config) for k, v in row.items()}
 
 
 def contract_uri(uristr: str, config: EndpointConfig) -> str:
+    """Contract URI. For e.g. "http://purl.org/dc/terms/" => "dc".
+
```

Comment:
what does it return if it's unable to replace the URI with a prefix? does it error? return none? return the URI again?

Reply:
@matentzn

Reply:
These aren't really philosophical questions, I'd suggest reading the code itself and reasoning about what it does directly. The answer seems pretty obvious from my side, but I think it's a good exercise for you to try yourself too